app/Http/Controllers/MemberController.php

```diff
@@ -38,17 +36,26 @@ public function update(StoreCreateMember $request)
         if ($memberEdit = Member::find($data['id'])) {
             $memberEdit->name = $data['name'];
             $memberEdit->phone_number = $data['phone_number'];
-            $memberEdit->information = $data['information'];
+            if (isset($data['information'])) {
+                $memberEdit->information = $data['information'];
+            } else {
+                $memberEdit->information ="null";
```

Comment:
I think this should be `null` not the string `"null"`. Also you have a code style error so please run `phpcs --standard=psr2 app tests database`